docs/proposals/scheduler/scheduler.proto

```diff
@@ -0,0 +1,81 @@
+syntax = "proto3";
+
+package scheduler;
+import "google/protobuf/empty.proto";
+
+enum Status {
+    RUNNING = 0;
+    STARTING = 1;
+    STOPPED = 2;
+    STOPPING = 3;
+    DESTROYING = 4;
+    TERMINATED = 5;
+    CRASHED = 6;
+    FAILED = 7;
+    SCHEDULING = 8;
+    SCHEDULED = 9;
+}
+
+enum Type {
+    CONTAINER = 0;
+}
+
+message Instance {
+    string id = 1;
+    string name = 2;
+    Type type = 3;
+    Status status = 4;
+    string uri = 5;
+    repeated string environnement = 6;
+    Resource resource = 7;
+    repeated string ports = 8;
+    string ip = 9;
+}
+
+message ResourceSummary {
+    int32 cpu = 1;
+    int32 memory = 2;
+    int32 disk = 3;
+}
+
+message Resource {
+    ResourceSummary max = 1;
+    ResourceSummary usage = 2;
```

Comment:
Please document what max vs usage means. Is max a limit or an actually seen, maximum usage?